BoxContentSDK/BoxContentSDK/BOXURLSessionCacheClient.h

```diff
@@ -280,6 +283,17 @@
  */
 - (BOOL)isSessionTaskCompletedForUserId:(NSString *)userId associateId:(NSString *)associateId;
 
+/**
+ * Check if a background session is valid, i.e. exists for a userId
+ *
+ * @param userID            Id of user
+ * @param backgroundSessionID       Id of background session
+ *
+ * @return YES if valid, NO if not
+ */
+- (BOOL)isBackgroundSessionValidGivenUserID:(NSString * _Nonnull)userID
```

Comment:
I think that `Given` is less natural than `For` in these method names.

For this specific name, the idea of `valid` is a little strange. Maybe something like `isBackgroundSessionID:associatedWithUserID:` would be clearer?

Reply:
updated in follow-up commit